src/components/DatePicker/CalendarDialog.tsx

```diff
@@ -156,7 +179,7 @@ const CalendarDialog = ({
                 items={months}
                 label={monthLabel}
                 initialSelectedItem={months.find(
-                  i => i.value === String(month)
+                  i => i.value === String(selectedMonth)
```

Comment:
**question:** Not sure if we want to change the initial data (because the selected can change but `initialMonth` and `initialYear` not. What do you think?